modules/custom/openy_activity_finder/js/programs_search.js

```diff
@@ -11,17 +11,19 @@
   });
 
   Vue.component('sidebar-filter', {
-    props: ['title', 'id', 'options', 'default', 'type', 'expanded', 'hide_label'],
+    props: ['title', 'id', 'options', 'default', 'type', 'expanded', 'hide_label', 'config_settings'],
```

Comment:
We need to make **config_settings** variable self-explanatory.
It is better to rename it ExpanderSectionsConfig or similar.